public/bower_components/angular-bootstrap/.bower.json

```diff
@@ -26,7 +26,6 @@
     "commit": "c4d2e80dbbe3df24e0c72d84dda34a35b0a81953"
```

Comment:
@tremainebuchanan exclude all `bower_components` files and folders from the repo.  I believe the command is:
```
git rm -r  --cached bower_components/
```